### PR TITLE
Script regression test

### PR DESCRIPTION
**Pull request by bachlx-bitmark**
update script delete photo before run test case

### CODE REVIEW

File: __tests__/common/common.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ const createNewAccountWithTouchId = async (driver) => {
         await driver
             .waitForElementByName('Allow', TEST_CONFIG.CHANGE_SCREEN_TIMEOUT)
             .elementByName("Allow").tap();
-    } catch {
+    } catch (error) {
```
Comment (phieudu241) [inline __tests__/common/common.js]: What is the purpose of adding `error` variable?

File: __tests__/utils/delete-simulator-photos.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const simctl = require('node-simctl');
+const { RUN_CONFIG } = require("../configs/config");
+const { runCommand, delay } = require("../common/common");
+
+
+simctl.getDevices(RUN_CONFIG.platformVersion, RUN_CONFIG.platformName).then(async (devices) => {
+  for (let device of devices) {
+    if (device.name === 'iPhone X') {
```
Comment (phieudu241) [inline __tests__/utils/delete-simulator-photos.js]: Don't see code for other iPhone types

File: __tests__/common/common.js (hunk 1 above)
Comment (bachlx-bitmark) [reply __tests__/common/common.js]: it require when we run directly by node

File: jest.config.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
     "<rootDir>/__tests__"
   ],
   "testMatch": [
-    "**/?(*.)+(spec|test).[jt]s?(x)"
+    "**/photo.test.js"
```
Comment (phieudu241) [inline jest.config.json]: Should revert to old config

File: __tests__/utils/delete-simulator-photos.js (hunk 2 above)
Comment (bachlx-bitmark) [reply __tests__/utils/delete-simulator-photos.js]: done

File: jest.config.json (hunk 3 above)
Comment (bachlx-bitmark) [reply jest.config.json]: done

**Pull request merged**